foreman-selinux-enable

```diff
@@ -13,15 +14,19 @@ trap "rm -rf '$TMP'" EXIT INT TERM
 for selinuxvariant in targeted
 do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
+    /usr/sbin/semanage port -E > $TMP_PORTS
+
+    # Remove previously defined elasticsearch_port_t
+    # (this can be removed in future release)
+    grep elasticsearch_port_t $TMP_PORTS | \
+      sed s/-a/-d/g >> $TMP
+
     # Load policy
     /usr/sbin/semanage module -S $selinuxvariant \
       -a /usr/share/selinux/${selinuxvariant}/foreman.pp.bz2
 
     echo "boolean -m --on httpd_setrlimit" > $TMP
 
-    /usr/sbin/semanage port -E | grep -q elasticsearch_port_t || \
-      echo "port -a -t elasticsearch_port_t -p tcp 9200-9300" >> $TMP
-
     /usr/sbin/semanage port -E | grep -q docker_port_t || \
```

Comment:
Note to myself - I will add TMP_PORTS for all statements, including the one that will appear after I rebase this on top of `develop`.